packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -455,9 +455,11 @@ export class ProjectModel {
     }
 
     async delete(projectUuid: string): Promise<void> {
-        await this.database('projects')
-            .where('project_uuid', projectUuid)
-            .delete();
+        await this.database.transaction(async (trx) => {
+            await trx('projects')
+                .where('project_uuid', projectUuid)
+                .delete();
```

Comment:
patch to fix prettier
```suggestion
            await trx('projects').where('project_uuid', projectUuid).delete();
```